invenio_communities/views/communities.py

```diff
@@ -98,12 +102,15 @@ def communities_settings(community=None, logo=None, pid_value=None):
         },
         # Pass permissions so we can disable partially UI components
         # e.g Settings tab
-        permissions=community.has_permissions_to(['update']),
+        permissions=community.has_permissions_to(
+            ['update', 'read', 'search_requests', 'search_invites']
+        ),
         active_menu_tab="settings"
     )
 
 @pass_community
 @pass_community_logo
+@require_community_search_requests
```

Comment:
I am not sure it is better to create decorators or simply see the code in the view for the permission checking.
It looks to me that it is quite some extra code, and I don't know how much it will be reused.